src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -204,6 +204,12 @@ public static void CopyToCovariantArray()
         [InlineData((4L * 1024L * 1024L * 1024L) + 256)]
         public static void CopyToLargeSizeTest(long bufferSize)
         {
+            // If this test is run in a 32-bit process, the large allocation will fail.
+            if (Unsafe.SizeOf<IntPtr>() != sizeof(long))
```

Comment:
I guess it's a subtle distinction. What you want to know is whether the pointer size is 8 bytes or larger, because you are making a large memory allocation which might fail otherwise. Comparing it to the size of another type feels a little roundabout to me, when you really just want to know if it is 8 bytes or larger.